docs/mp/openapi/openapi.adoc

```diff
@@ -111,11 +111,11 @@ Helidon MP QuickStart, enhanced with OpenAPI support.
     description = "Conveys the new greeting prefix to use in building greetings",
     content = @Content(
                 mediaType = "application/json",
-                schema = @Schema(implementation = GreetingMessage.class),
+                schema = @Schema(implementation = GreetingUpdateMessage.class),
                 examples = @ExampleObject(
                     name = "greeting",
                     summary = "Example greeting message to update",
-                    value = "New greeting message")))
+                    value = "{\"greeting\": \"New greeting message\"")))
```

Comment:
I think the syntax should be
```
value = "{\"greeting\": \"New greeting message\"}")))
```
to include the closing curly brace.

Reply:
Oooppss)).  I'm blind).